ir/ir.rs

```diff
@@ -513,6 +513,15 @@ pub enum ItemKind {
 
     /// Miscellaneous item. Perhaps metadata. Perhaps something else.
     Misc(Misc),
+
+    /// Program scope. Modules, namespaces, imported declarations.
+    Scope(Scope),
```

Comment:
Scopes only fuzzily exist within a binary, and I'm not sure I see how they are useful for size profiling. Can you explain some more how this is intended to be used?